BOOKBBANG/BOOKBBANG/Views/Book/DetailBook/BookInfo.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  BookInfo.swift
+//  BOOKBBANG
+//
+//  Created by Jerrie on 8/1/24.
+//
+
+import SwiftUI
+
+struct BookInfo: View {
+    var author: String = "히가시노 게이고(지은이), 최고은(옮긴이)"
+    var genre: String = "소설"
+    var name: String = "당신이 누군가를 죽였다"
+    var phraseCount: Int = 100
+    var publisher: String = "제리"
+    var readDate: Date = Date()
+    var formattedDate: String {
+        let calendar = Calendar.current
+        let components = calendar.dateComponents([.year, .month, .day], from: readDate)
+        
+        if let year = components.year, let month = components.month, let day = components.day {
+            return "\(year). \(month). \(day)"
+        } else {
+            return ""
+        }
+    }
+    
+    var readStatus: ReadStatus = .readIng
+    
+    var body: some View {
+        HStack {
+            VStack(alignment: .leading, spacing: 39) {
+                VStack(alignment: .leading, spacing: 8) {
+                    Text("\(name)")
+                        .font(.bookTitle)
+                        .foregroundStyle(.typo100)
```

Comment:
<img width="222" alt="image" src="https://github.com/user-attachments/assets/f4de090b-570e-45a7-ac09-f4918ef9a5c6">

kerning 넣어주세용! 
+ 문자열 보간법 안써도 String이라서 괜찮습ㄴㅣ다.

```suggestion
                    Text("name")
                        .font(.bookTitle)
                        .foregroundStyle(.typo100)
                        .kerning(-1)
```